internal/command/diff_inputs.go

```diff
@@ -0,0 +1,390 @@
+package command
+
+import (
+	"fmt"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/simplesurance/baur/v1"
+	"github.com/simplesurance/baur/v1/internal/digest"
+	"github.com/simplesurance/baur/v1/internal/format"
+	"github.com/simplesurance/baur/v1/internal/format/csv"
+	"github.com/simplesurance/baur/v1/internal/format/table"
+	"github.com/simplesurance/baur/v1/storage"
+)
+
+type storageInput struct {
+	input *storage.Input
+}
+
+func (i *storageInput) Digest() (*digest.Digest, error) {
+	return digest.FromString(i.input.Digest)
+}
+
+func (i *storageInput) String() string {
+	return i.input.URI
+}
+
+type differenceType int
+
+const (
+	digestMismatch differenceType = iota
+	existsInOneOnly
+	existsInTwoOnly
+)
```

Comment:
The first List of Inputs is the base of comparison, therefore we could name the difference types like:

```suggestion
const (
	digestMismatch differenceType = iota
	removed
	added
)
```